### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by Divyesh85** [gettek/terraform-azurerm-policy-as-code#30] [Q&A] [answered]
Hi gettek, 

I was checking out your repository for terraform-azurerm-policy-as-code , i get the basic understanding about how to use the modules  but wanted to know if i don't have management group created for the policies, can i still create the policies based on the subscription id? Do i need to make more changes to code to achieve that or your example should work without much changes?

**Comment by gettek** [comment, marked as answer]
Hi @Divyesh85,

When you omit the management group ID, Terraform will create the definition/assignment at the current authenticated subscription scope or to target multiple subscriptions try using [provider alias's ](https://www.terraform.io/language/providers/configuration#alias-multiple-provider-configurations)

**Reply by Divyesh85** [reply]
HI @gettek ,
Thanks for replying , i was indeed able to make it work on subscription level and is working as expected some time back. One more thing i wanted to know if there is any examples on policies related to enforce casing or add created_at and updated_at for tags.  If you have something for this let me know.

**Reply by gettek** [reply]
Unfortunately I don't but this could be possible with a Modify effect Policy.

There are also some [Community Samples available here](https://github.com/Azure/Community-Policy/tree/master/Policies/Tags)
